### PR TITLE
Bk/issue 868 number textbox accepting alphabets

### LINKED ISSUE #868: On Add patient, Phone number textbox accepting alphabets
**Describe the bug**
On Add patient, Phone number textbox accepting alphabets, It should accepts only numbers and some special characters

**Component**
SDC library, reference app

**To Reproduce**
1. Go to 'Reference App'
2. Click on '+' icon to add patient
3. Enter First Name, family name, Date of birth
4. Tap on phone number, Observe keyboard
5. Enter Alphabets and click on Submit button
6. Observe behaviour

**Expected behavior**
Phone number should accept numbers and some special character

**Screenshots**
Screenshot_20211026-131415

**Smartphone (please complete the following information):**
Device: Nokia 3.4
Android version: 11
Build number: 00WW_2_410

### PR DESCRIPTION
**Pull request by BishyanKar**
**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straightforward changes).**

Fixes #868 

**Description**
Line 291 changed inside new-patient-registration-paginated.json file so as to prevent phone number edit text from accepting alphabets.

**Alternative(s) considered**
NA

**Type**
Bug fix

**Screenshots (if applicable)**
After fix
FHIR issue 868

**Checklist**
- [x] I have read and acknowledged the [Code of conduct](https://github.com/google/android-fhir/blob/master/docs/CODE_OF_CONDUCT.md)
- [x] I have read [How to Contribute](https://github.com/google/android-fhir/blob/master/docs/contributing.md)
- [x] I have read the [Developer's guide](https://github.com/google/android-fhir/wiki/Developer's-Guide)
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate )
- [ ] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project
- [x] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally
- [x] I have built and run the reference app(s) to verify my change fixes the issue and/or does not break the reference app(s)

### CODE REVIEW

**Comment by BishyanKar** [comment]
@jingtang10 @aditya-07  can any of you please approve the workflow?

File: reference/src/main/assets/new-patient-registration-paginated.json [hunk 1]
```diff
@@ -288,7 +288,7 @@
               ],
               "linkId": "PR-telecom-value",
               "definition": "http://hl7.org/fhir/StructureDefinition/Patient#Patient.telecom.value",
-              "type": "string",
+              "type": "decimal",
```
Comment (santosh-pingle) [inline reference/src/main/assets/new-patient-registration-paginated.json:291]: can we try `integer` here?

Comment (BishyanKar) [reply reference/src/main/assets/new-patient-registration-paginated.json:291]: I tried having integer but it doesn't accept more than 9 digits.
FHIR issue 868 (2)
s

Comment (epicadk) [reply reference/src/main/assets/new-patient-registration-paginated.json:291]: Will this not cause issues for the country code?

Comment (santosh-pingle) [reply reference/src/main/assets/new-patient-registration-paginated.json:291]: yes we discussed about the country code when issue was raised, and what I remember we thought not to think/consider about country code at the moment... 
cc aditya-07, jingtang10

Comment (epicadk) [reply reference/src/main/assets/new-patient-registration-paginated.json:291]: Why aren't we using the `phone` input type?

Comment (BishyanKar) [reply reference/src/main/assets/new-patient-registration-paginated.json:291]: I thought of adding a separate type for phone but the Questionnaire.java file is locked.

Comment (santosh-pingle) [reply reference/src/main/assets/new-patient-registration-paginated.json:291]: https://github.com/google/android-fhir/pull/838

Comment (jingtang10) [reply reference/src/main/assets/new-patient-registration-paginated.json:291]: > 

There's no Questionnaire.java in the codebase. Everything is in Kotlin. Is this some local artifact from your local build?

**Comment by jingtang10** [comment]
Thanks for the PR but unfortunately this is not the right solution. We can't just use decimal type for phone numbers. Please see #838. I'll close this for now.

**Pull request closed without merging**